src/pykeen/triples/triples_factory.py

```diff
@@ -589,6 +589,47 @@ def new_with_restriction(
             extra_metadata=extra_metadata,
         )
 
+    def to_path_binary(
+        self,
+        path: Union[str, pathlib.Path, TextIO],
+    ) -> None:
+        """
+        Save triples factory to path in (PyTorch's .pt) binary format.
+
+        :param path:
+            The path to store the triples factory to.
+        """
+        path = normalize_path(path)
+        torch.save(dict(
```

Comment:
is torch's save any more than a wrapper around pickling? maybe we should also go as far as to use gzip to compress it, since pickle is pretty verbose

Reply:
`torch` uses a zipfile since 1.6, cf. [here](https://pytorch.org/docs/stable/generated/torch.save.html#torch.save)

> The 1.6 release of PyTorch switched torch.save to use a new zipfile-based file format. torch.load still retains the ability to load files in the old format. If for any reason you want torch.save to use the old format, pass the kwarg _use_new_zipfile_serialization=False.